advanced.md

```diff
@@ -167,4 +167,112 @@ public void toggleAllDigitalPins( bool setPinsHigh )
 		firmata.sendDigitalPort( 1, 0 );	//all pins low
```

Comment:
Magic numbers are ill-advised. Constants disambiguate states, and should be used in favor of a specific interpretation of the value. If your concern is about conflagrating, introduce a new value for ALLLOW.